plugins/zen-bit/includes/class-zen-bit-api.php

```
@@ -309,15 +309,21 @@ private static function filter_by_date(array $events, string $ds, string $de): a
+            $start_ts = strtotime((string) ($e['starts_at'] ?? ''));
+            $end_ts = strtotime((string) ($e['ends_at'] ?? $e['end_date'] ?? ''));
+            $lower_bound_ts = $end_ts ?: $start_ts;
+            $upper_bound_ts = $start_ts ?: $end_ts;
+
+            if (!$lower_bound_ts && !$upper_bound_ts) {
                 return false;
             }
-            $d = \gmdate('Y-m-d', $ts);
-            if ($ds !== '' && $d < $ds) {
+
+            $lower_bound_date = \gmdate('Y-m-d', $lower_bound_ts);
```

PHPStan (2.2.2)

[warning] Parameter `#2` $timestamp of function gmdate expects int|null, int|false given.


(argument.type)